frontend/app/components/people-skill-new.js

```diff
@@ -34,10 +34,23 @@ export default Component.extend({
       skills = skills.filter(
         skill => !peopleSkillsIds.includes(skill.get("id"))
       );
-      return skills.sort((a, b) => (a.get("title") < b.get("title") ? -1 : 1));
+      return skills.sort((a, b) => this.orderByTitleCaseInsensitive(a, b));
     });
   }),
 
+  orderByTitleCaseInsensitive(firstSkill, secondSkill) {
+    let firstTitle =
+      firstSkill.get("title") == undefined
+        ? undefined
+        : firstSkill.get("title").toLowerCase();
+    let secondTitle =
+      secondSkill.get("title") == undefined
+        ? undefined
+        : secondSkill.get("title").toLowerCase();
+
+    return firstTitle < secondTitle ? -1 : 1;
```

Comment:
Replace this with three return statements
```
  if (firstTitle < secondTitle) {
    return -1;
  }
  if (secondTitle > firstTitle) {
    return 1;
  }
  return 0
```
This way, you also cover the case of them being the same string. It's also just like in the example on: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort